packages/sucrase/types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Plugin } from 'rollup';
+type Options = {
+  transforms?: string[];
+  include?: string[];
+  exclude?: string[];
```

Comment:
these should use `FilterPattern` https://github.com/rollup/plugins/blob/ca0540f30ef3afd5bd998132e6b5931b0c319234/packages/pluginutils/types/index.d.ts#L23